services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/helpers/KibanaHelper.groovy

```diff
@@ -0,0 +1,111 @@
+/* Copyright 2018 Telstra Open Source
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License");
+ *   you may not use this file except in compliance with the License.
+ *   You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   Unless required by applicable law or agreed to in writing, software
+ *   distributed under the License is distributed on an "AS IS" BASIS,
+ *   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *   See the License for the specific language governing permissions and
+ *   limitations under the License.
+ */
+
+package org.openkilda.functionaltests.helpers
+
+import groovy.util.logging.Slf4j
+import org.openkilda.testing.service.elastic.ElasticHelper
+
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.http.HttpEntity
+import org.springframework.http.HttpHeaders
+import groovy.json.JsonBuilder
+import org.springframework.stereotype.Component
+
+/**
+ * A set of helper methods to access, read and filter log entries from the log management system
+ */
+
+@Component
+@Slf4j
+class KibanaHelper {
+
+    @Autowired
+    private ElasticHelper elasticHelper
+
+    /**
+     * Searches Elastic Search database for a specific log entries
+     * @param app_id - application ID to lookup (either app_id or tags should be speficied)
+     * @param tags - one or more tag, delimited by OR operator, to lookup (either app_id or tags should be specified)
+     * @param level - log level (can be null)
+     * @param timeRange - search depth (in seconds, set to 0 to search all).
+     * @param resultCount - max number of returned documents (100 default)
+     * @param defaultField - field in log entry to search at (_source by default)
+     * @param index - Elastic Search index to lookup (_all by default)
+     *
+     * In case you need to lookup multiple application IDs, tags or log levels, pass parameters as:
+     * "VALUE1 OR VALUE2 OR ... OR VALUE_N"
+     */
+    def getLogs(String app_id=null, String tags=null, String level="INFO OR WARN OR ERROR", Map<String,
```

Comment:
Too many input params. Can we take a certain object with required data as input? I would rather do `getLogs(new LogsRequest(level: "INFO", timeRange: 30))`

Reply:
Added a builder-like interface that accepts Closure to add all required parameters.
Also, added a simplified interface that omits most of the unnecessary data.